tests/testflows/map_type/configs/clickhouse/config.d/logs.xml

```diff
@@ -0,0 +1,17 @@
+<yandex>
+    <shutdown_wait_unfinished>3</shutdown_wait_unfinished>
```

Comment:
Why?

Reply:
https://github.com/ClickHouse/ClickHouse/blob/d305b233389fcd0c044d4e291286455fd6e3ea6b/programs/server/Server.cpp#L920. The default seems to be 5 sec so you are right we can remove this.

Reply:
Removed.